### PR TITLE
Imp/add card actions

### PR DESCRIPTION
**Pull request by patricksimonian**
# About
First of few additions to the card layout

## Changes
- added the `<Ribbon />` and `<ActionsRibbon />` component

### CODE REVIEW

File: app-web/src/constants/ariaLabels.js [hunk 1]
```diff
@@ -4,3 +4,6 @@ export const ARIA_LABEL_FILE_IN_REPO =
 export const ARIA_LABEL_RESOURCE = 'Learn more about this article';
 export const ARIA_LABEL_FILTER_SELECT = 'Select to filter cards by the resource type';
 export const ARIA_LABEL_TO_GITHUB_USER = 'Select to view this authors Github Profile';
+export const ARIA_LABEL_TO_GITHUB_ISSUES = "Select to view this respositories' issues";
```
Comment (ShellyXueHan) [inline app-web/src/constants/ariaLabels.js]: better to make descriptions more details by saying 'this Github repos....'

File: app-web/src/constants/ui.js [hunk 2]
```diff
@@ -31,36 +69,16 @@ export const CARD_CONFIG = {
...
-    {
-      name: 'Watch',
-      flag: 'features.card.watch',
-      icon: faEye,
-      ariaLabel: ARIA_LABEL_WATCH_GITHUB_REPO,
-      getHref: getGithubWatchRoute,
-    },
-  ],
+  actionsRibbon: {
+    [RESOURCE_TYPES.RESPOSITORIES]: [
```
Comment (ShellyXueHan) [inline app-web/src/constants/ui.js]: why do you need the []?

File: app-web/src/constants/ariaLabels.js (hunk 1 above)
Comment (patricksimonian) [reply app-web/src/constants/ariaLabels.js]: changed!

File: app-web/src/constants/ui.js (hunk 2 above)
Comment (patricksimonian) [reply app-web/src/constants/ui.js]: Oh i'm setting the key to be exactly the same as the enum for resource types
that way if resource types are ever changed this config doesn't need to be changed

**Pull request merged**